udp_driver/CMakeLists.txt

```diff
@@ -37,16 +45,19 @@ if(Boost_FOUND)
   link_directories(${Boost_LIBRARY_DIRS})
 endif()
 
-include_directories(include)
-
 ament_auto_add_library(${PROJECT_NAME} SHARED
   src/udp_socket.cpp
   src/udp_driver.cpp
-  examples/udp_driver_node.cpp)
+  src/udp_driver_node.cpp)
 ament_target_dependencies(${PROJECT_NAME} "Boost")
 
 target_compile_options(${PROJECT_NAME} PUBLIC "-O0" PRIVATE -Wno-sign-conversion -Wno-conversion)
 
+#rclcpp_components_register_node(udp_driver_node
+  #PLUGIN "drivers::udp_driver::UdpDriverNode"
+  #EXECUTABLE ${PROJECT_NAME}_node_exe
+#)
+
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
```

Comment:
same thing here, do we need the `test/gtest_main.cpp` similar to what we did within `io_context`?

Reply:
We don't. Removed here and removed the actual file.